src/Soap/SoapBase.php

```diff
@@ -533,15 +533,15 @@ public function saveTemporarilyKeyFiles()
                 $this->temppass
             );
         }
-        $ret &= $this->filesystem->put(
+        $ret &= $this->filesystem->write(
```

Comment:
Olá Elvis!! Obrigado pelo PR! Versão 2 do thephpleague/flysystem não retorna mais boleano, leia https://flysystem.thephpleague.com/v2/docs/what-is-new/. Acredito que aqui devemos envolver em um `try catch` e capturar a exceção `FilesystemException`